apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -573,6 +573,8 @@ def tables(self) -> BlockwiseTableReadIter:
             size=self.size,
             reindex_disable_on_axis=self.reindex_disable_on_axis,
             eager=self.eager,
+            compress=True,
```

Comment:
`compress` is meaningless in this context - tables never have a compresses axis.  Why is the flag being added here?